src/chainparams.cpp

```diff
@@ -561,6 +563,7 @@ class CDevNetParams : public CChainParams {
         consensus.DIP0003EnforcementHeight = 2; // DIP0003 activated immediately on devnet
         consensus.DIP0003EnforcementHash = uint256();
         consensus.DIP0008Height = 2; // DIP0008 activated immediately on devnet
+        consensus.MinBIP9WarningHeight = consensus.DIP0008Height + consensus.nMinerConfirmationWindow;
```

Comment:
why should this be dip0008? Also, should be this be bumped periodically / with each new hf hardening?